### ISSUE TITLE
Allow underlying entities to follow Auto mode changes

### ISSUE THREAD

**Issue by paulbyrne-spacenk** [tetele/hvac_group#79]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/tetele/hvac_group/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

My heating entities have their own internal schedule system, which is configurable via their own app or a custom component in HA. It changes the target temp based on time and sets them to a low temp If everyone is away. If I add them to this helper, the helper and heating entities get out of sync and either end up fighting or it just kinda breaks.

### Describe the solution you'd like

Ideally, changes to the underlying entities should propagate back up to the helper. Eg. Setting the temp on the helper should set the temp on the entity, AND setting the temp on the entity should set the temp on the helper. That way you could group scheduled heating and dumb AC into one helper.

### Describe alternatives you've considered

Tried climate group, but it isn't really for this sort of thing. Your helper is the closest, it just does quite handle automatic temp adjustment from the underlying entities.

### Additional context

N/A

**Comment by paulbyrne-spacenk** [comment]
This is sort of related to #67

**Comment by tetele** [comment]
I think it's the same issue, not just related. Is there any difference you're seeing? If not, do you mind if I close it?

**Comment by paulbyrne-spacenk** [comment]
Yeah, I guess we are both asking for two way syncing of states, just for different reasons. Go ahead and mark it as a dupe.
